src/test/kotlin/no/nav/veilarbaktivitet/mock_nav_modell/NavMockService.kt

```diff
@@ -12,8 +13,8 @@ class NavMockService(
     val oppfolgingsperiodeService: OppfolgingsperiodeService,
 ) {
 
-    fun createHappyBruker(brukerOptions: BrukerOptions = BrukerOptions.happyBruker()): MockBruker {
-        val bruker = MockNavService.createBruker(brukerOptions)
+    fun createHappyBruker(brukerOptions: BrukerOptions = BrukerOptions.happyBruker(), fnr: Fnr? = null): MockBruker {
```

Comment:
Her ser jeg for meg at man kunne beholdt tidligere funksjon men lagt til en ny med denne nye signaturen. Altså to funksjoner: 
- `createHappyBruker(BrukerOptions)`
- `createHappyBruker(BrukerOptions, Fnr)`

Den første kunne da ha kalt på den siste, og der man i Java nå skriver `createHappyBruker(brukerOptions, null)` så kunne man endre det til å bare skrive `createHappyBruker(brukerOptions)`.